crates/bevy_reflect/bevy_reflect_derive/src/derive_data.rs

```diff
@@ -37,6 +41,8 @@ pub(crate) struct ReflectMeta<'a> {
     generics: &'a Generics,
     /// A cached instance of the path to the `bevy_reflect` crate.
     bevy_reflect_path: Path,
+    /// A collection corresponding to `ignored` fields' indices during serialization.
+    serialization_blacklist: BitSet<u32>,
```

Comment:
Imo this shouldn't be in `ReflectMeta` since it's not relevant to non-struct-like types. I get why you did it, though. Maybe we instead put this in `ReflectStruct` directly and add a `get_type_registration` method to it. Then we should use `ReflectStruct::get_type_registration` over `ReflectMeta::get_type_registration` for all structs.

Reply:
Yeah originally this was also applicable to enums when I rebased, but since it's no longer relevant I've moved this as proposed